src/index.ts

```diff
@@ -2,11 +2,19 @@
  * This file generates a server instance
  */
 import dotenv from "dotenv";
+import path from "path";
 
-dotenv.config();
-const PORT = process.env.SERVER_PORT;
 import app from "./app";
 
+dotenv.config();
```

Comment:
doing this would not expose these environment variables in testing, as test runner is only including the app.